packages/ui-components/src/components/ComboBoxOption/ComboBoxOption.component.tsx

```diff
@@ -91,10 +99,10 @@ export const ComboBoxOption = ({ children, disabled = false, value = "", label,
   )
 }
 
-ComboBoxOption.propTypes = {
-  children: PropTypes.string,
-  disabled: PropTypes.bool,
-  value: PropTypes.string,
-  label: PropTypes.string,
-  className: PropTypes.string,
-}
+export interface ComboBoxOptionProps extends React.HTMLProps<HTMLLIElement> {
+  children?: string
```

Comment:
(Improvement Question, Not Blocking)

Do you know why we use `string` here for `children`? Not suggesting to change now or anything, but we usually use `ReactNode`. Is this somehow special?

Reply:
good question

Reply:
In proptypes it used to be string, so I moved it to TS as string. I think it was for the reason that combobox option text was meant to be string rather than arbitrary HTML.

Reply:
Resolving this thread